src/quota_aggregator_impl.cc

```diff
@@ -125,7 +129,8 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::Quota(
     // requests will be aggregated to this temporary element until the
     // response for the actual request arrives.
     ::google::api::servicecontrol::v1::AllocateQuotaResponse temp_response;
-    CacheElem* cache_elem = new CacheElem(temp_response);
+    CacheElem* cache_elem = new CacheElem(request, temp_response,
+                                          SimpleCycleTimer::Now());
     cache_elem->set_signature(request_signature);
     cache_elem->set_in_flight(true);
     cache_->Insert(request_signature, cache_elem, 1);
```

Comment:
Here even for the first time (new cache item,   we can just return temp_response,  and use AddRemovedItem() to send the refresh to sever. 

By doing this way, even the first request, we are not waiting for server respond